### PR TITLE
AEROGEAR-2480 set up antora structure

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@laurafitzgerald this is my idea for documenting processes. 
just an outline for now. If you accept PR I can build it and push to shiftgear.org for review,

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+= AeroGear Processes
+
+image::aerogear_logo_200px.png[]
+
+This documentation covers the AeroGear processes relating to:
+
+* User Feedback
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/index.adoc:7]: So there could be an additional heading here?

**Comment by laurafitzgerald** [comment]
@finp I took a quick look. To me it looks like it's mostly skeleton setup stuff which I'm assuming is fine. If there are steps to verify this locally (if that's possible) could you post them here. 

Adding content will probably require a more extensive review. The discussion that kicked off this repo was to capture (among other things) our agile processes, i.e. how we work as a team to get work agreed through a team and completed. This also has some considerations for community as there could be requests for changes coming through the community. Do you think this warrants a separate section or do you think it will fit under community? cc @lgriffin you might have some thoughts here from the original discussion. 
I would be leaning towards having a separate section.

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (pwright) [reply modules/ROOT/pages/index.adoc:7]: @laurafitzgerald  we can add whatever we want here, as you said the intention here is to just set up a skeleton

**Comment by pwright** [comment]
created #2

**Comment by lgriffin** [comment]
I'm happy with the skeleton and can decide on content and what our Agile processes structure looks like then

**Comment by pwright** [comment]
@laurafitzgerald  you can validate this per instructions at http://shiftgear.org/processes/latest/docs/antora.html

**Pull request merged**